apps/site/src/pages/wordpress.page.tsx

```diff
@@ -12,6 +13,7 @@ const WordPressPage: FunctionComponent = () => (
     <WhyInstall />
     <WhyDevelop />
     <Availability />
+    {/* <FooterBanner /> */}
```

Comment:
Is this comment / import intentional?

Reply:
Removed in 52e7b38657ed3503cbc9cd63346ec90bc7620e24